src/common/header/Header.tsx

```diff
@@ -71,6 +72,13 @@ function Header(): React.ReactElement {
     }
   };
 
+  // List all languages as primary languages
+  const sortLanguageOptions: LanguageSelectorProps['sortLanguageOptions'] = (
+    options: LanguageOption[],
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    _activeLanguage
```

Comment:
I think this can be happily left out since it is not used? 🤔

Reply:
yes, fix:ed